### PR TITLE
GAWB-982: Write a test that makes sure we're sending the right JSON to Cromwell for list-y workflow inputs.

### PR DESCRIPTION
**Pull request by skwalker**
I based what the input should look like off of [this](https://github.com/broadinstitute/cromwell/blob/develop/engine/src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala#L273) test
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: If updating admin endpoints, also update [firecloud-admin-cli](https://github.com/broadinstitute/firecloud-admin-cli)
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: JIRA ticket checks:
  - Acceptance criteria exists and is met
  - Note any changes to implementation from the description
  - Add notes on what you've tested
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Database checks:
  - If PR includes new or changed db queries, include the explain plans in the description
  - Make sure liquibase is updated if appropriate
  - If doing a migration, take a backup of the dev and alpha DBs in Google Cloud Console
- [x] **Submitter**: Update FISMA documentation if changes to:
  - Authentication
  - Authorization
  - Encryption
  - Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] Tell  that the PR exists if he wants to look at it **(apply requires_doge label)**
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- Review cycle:
  - LR reviews
  - Rest of team may comment on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH
  - Submitter updates documentation as needed
  - Submitter **reassigns to LR** for further feedback
- [ ]  sign off
- [x] **LR** sign off
- [ ] **Assign to submitter** to finalize
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [ ] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala [hunk 1]
```diff
@@ -306,4 +310,77 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
     awaitCond(runAndWait(workflowQuery.findWorkflowByIds(workflowRecs.map(_.id)).map(_.status).result).forall(_ == WorkflowStatuses.Failed.toString), 10 seconds)
     workflowSubmissionActor ! PoisonPill
   }
+
+  it should "not truncate array inputs" in withDefaultTestDatabase {
+
+    mockServer.stopServer
```
Comment (skwalker) [inline src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala]: Had to do this to make the tests work properly in sbt -- there's something weird going on with the mockserver where other tests influence what happens with this.

File: src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala [hunk 2]
```diff
@@ -27,7 +31,7 @@ class WorkflowSubmissionSpec(_system: ActorSystem) extends TestKit(_system) with
 
   def this() = this(ActorSystem("WorkflowSubmissionSpec"))
   val mockGoogleServicesDAO: MockGoogleServicesDAO = new MockGoogleServicesDAO("test")
-  val mockServer = RemoteServicesMockServer()
+  var mockServer = RemoteServicesMockServer()
```
Comment (dvoet) [inline src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala]: I don't like making this a var. Instead make the mock server a constructor parameter to TestWorkflowSubmission.

Comment (skwalker) [reply src/test/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionSpec.scala]: so then for each test we would have to start and stop the mock server?

**Comment by dmohs** [comment]
👍

**Comment by dvoet** [comment]
👍

**Pull request merged**